sn_client/src/api.rs

```diff
@@ -144,13 +151,13 @@ impl Client {
                 }
                 Ok(ClientEvent::InactiveClient(timeout)) => {
                     let random_target = ChunkAddress::new(XorName::random(&mut rng));
-                    debug!("No ClientEvent activity in the past {timeout:?}, performing a random get_chunk query to target: {random_target:?}");
+                    debug!("No ClientEvent activity in the past {timeout:?}, performing a random get_store_cost query to target: {random_target:?}");
```

Comment:
This log line has to be updated as we do not perform random queries on inactivity anymore.